.reuse/dep5

```diff
@@ -0,0 +1,23 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: Astarte Device SDK ESP32
+Source: https://github.com/astarte-platform/astarte-device-sdk-esp32
+
+Files: doc/docmain.dox
+Copyright: 2023 SECO Mind Srl
+License: Apache-2.0
+
+Files: examples/**/README.md
```

Comment:
Ideally we can reduce to the bare minimum the number of listed files here:
e.g. we can do the following on `README.md` files:
```md
<!---
  Copyright 2023 SECO Mind Srl

  SPDX-License-Identifier: LGPL-2.1-or-later or Apache-2.0
-->
```